concepts/05 Widgets/Scheduler/070 View Switcher.md

```diff
@@ -37,6 +37,58 @@ To specify what views are available within the switcher, use the [views](/api-re
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :views='views'
+		/>
```

Comment:
Please use a text editor that prints 4 spaces when you press Tab or press Space instead of Tab.